include/verilated_trace.h

```diff
@@ -48,6 +48,11 @@ class VerilatedTraceBuffer;
 template <class T_Buffer>
 class VerilatedTraceOffloadBuffer;
 
+// Traced strings above this size may be inefficient as they cannot be deduped.
+// Must be <=254, but larger sizes bloat memory proportional to the number of
+// traceable strings.
+constexpr size_t LEN_FAST_TRACED_STRING = 63;
+
```

Comment:
Please put in verilatedos.h after VL_VALUE_STRING_MAX_CHARS so you don't need to include this in src, and use a define instead.